src/client/power-pages/actions-hub/ActionsHubTreeDataProvider.ts

```diff
@@ -6,40 +6,59 @@
 import * as vscode from "vscode";
 import { ActionsHubTreeItem } from "./tree-items/ActionsHubTreeItem";
 import { OtherSitesGroupTreeItem } from "./tree-items/OtherSitesGroupTreeItem";
-import { ITelemetry } from "../../../common/OneDSLoggerTelemetry/telemetry/ITelemetry";
 import { Constants } from "./Constants";
 import { oneDSLoggerWrapper } from "../../../common/OneDSLoggerTelemetry/oneDSLoggerWrapper";
+import { PacTerminal } from "../../lib/PacTerminal";
+import { EnvironmentGroupTreeItem } from "./tree-items/EnvironmentGroupTreeItem";
+import { IEnvironmentInfo } from "./models/IEnvironmentInfo";
+import { authManager } from "../../AuthManager";
 
 export class ActionsHubTreeDataProvider implements vscode.TreeDataProvider<ActionsHubTreeItem> {
     private readonly _disposables: vscode.Disposable[] = [];
     private readonly _context: vscode.ExtensionContext;
-    private readonly _telemetry: ITelemetry;
+    private readonly _pacTerminal: PacTerminal;
 
-    private constructor(context: vscode.ExtensionContext, telemetry: ITelemetry) {
+    private constructor(context: vscode.ExtensionContext, pacTerminal: PacTerminal) {
         this._disposables.push(
             vscode.window.registerTreeDataProvider("powerpages.actionsHub", this)
         );
 
         this._context = context;
-        this._telemetry = telemetry;
+        this._pacTerminal = pacTerminal;
     }
 
-    public static initialize(context: vscode.ExtensionContext, telemetry: ITelemetry): void {
-        new ActionsHubTreeDataProvider(context, telemetry);
-
-        telemetry.sendTelemetryEvent(Constants.EventNames.ACTIONS_HUB_INITIALIZED);
+    public static initialize(context: vscode.ExtensionContext, pacTerminal: PacTerminal): ActionsHubTreeDataProvider {
+        const actionsHubTreeDataProvider = new ActionsHubTreeDataProvider(context, pacTerminal);
         oneDSLoggerWrapper.getLogger().traceInfo(Constants.EventNames.ACTIONS_HUB_INITIALIZED);
+
+        return actionsHubTreeDataProvider;
```

Comment:
Any reason for changing the return type here?

Reply:
Was needed to get the instance of the tree while writing tests